README.md

```diff
@@ -1,3 +1,5 @@
+[![Build Status](https://travis-ci.org/expectedbehavior/acts_as_archival.svg?branch=master)](https://travis-ci.org/expectedbehavior/acts_as_archival)
```

Comment:
That gem version one is pretty good. We've use [scrutinizer](https://scrutinizer-ci.com/) in the past and I have positive feelings about it. For now, maybe just gem version and travis